bn_mp_prime_rabin_miller_trials.c

```diff
@@ -35,7 +35,7 @@ int mp_prime_rabin_miller_trials(int size)
          return (x == 0) ? sizes[0].t : sizes[x - 1].t;
       }
    }
-   return sizes[x-1].t + 1;
+   return sizes[x-1].t;
```

Comment:
Is this is the inconsistency? I cannot judge this since I never looked into those prime tests. Ping @czurnieden?